src/selectors/interface.js

```diff
@@ -72,26 +72,26 @@ export const makeNetworkNodesForSubject = () => {
 };
 
 export const makeNetworkNodesForPrompt = () => {
-  // used to check node attributes
-  const getIds = makeGetIds();
+  // used to check prompt ids
+  const getAttributes = makeGetAdditionalAttributes();
```

Comment:
Why the re-revision?

Maybe we should create a couple of new methods with names that better describe what they do?

Also, if the notable change has been reverted we can probably remove the comment.

Reply:
This related to using the attributes as a match for which nodes belonged. Since you could drag nodes from other prompts into the main panel, I was using the attributes instead of promptid.